### DISCUSSION TITLE
keeping-secrets-secret-with-git-crypt/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#235] [Announcements]
# keeping-secrets-secret-with-git-crypt/

Figuring out how to keep secrets secret using git-crypt and PGP keys.

https://blog.ktz.me/keeping-secrets-secret-with-git-crypt/

**Comment by clempat** [comment]
This looks nice. By using 1Password for GPG, I can use this without worrying about moving the GPG key. Do you always deploy from your computer? 

Another question: should we be concerned that this repository has not been updated in 3 years and has around 100 issues?

**Reply by ironicbadger** [reply]
Which repo hasn't been updated in 3 years?

**Reply by clempat** [reply]
Git-Crypt has not had a release in 3 years, and the last commit (aside from a version upgrade in GitHub action) is also 3 years old. This may be fine if it does not depend on any external dependencies that need updating. There is still issues open though.

**Comment by AlienResidents** [comment]
Consider moving to SOPS which is maintained, and can leverage multiple backends including gpg. https://github.com/getsops/sops

**Comment by quentinsf** [comment]
In case useful, we've been using Blackbox, from StackExchange, for several years.  Not highly polished, but it works well.  It's less automatic (though it has some knowledge of git) but it does also work with other version control systems, in the (admittedly somewhat unlikely now) case that you need that!

https://github.com/StackExchange/blackbox
